examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart

```diff
@@ -24,7 +24,7 @@ const List<Color> coolColors = const <Color>[
 const List<String> coolColorNames = const <String>[
   'Sarcoline', 'Coquelicot', 'Smaragdine', 'Mikado', 'Glaucous', 'Wenge',
   'Fulvous', 'Xanadu', 'Falu', 'Eburnean', 'Amaranth', 'Australien',
-  'Banan', 'Falu', 'Gingerline', 'Incarnadine', 'Labrabor', 'Nattier',
+  'Banan', 'Falu', 'Gingerline', 'Incarnadine', 'Labrador', 'Nattier',
```

Comment:
I'm not sure about this change

Reply:
The correction's right. I just took a bunch of random colors from http://mentalfloss.com/article/59328/19-colors-youve-probably-never-heard. But this is misspelled.